backend/handlers/like_handler.py

```diff
@@ -46,9 +46,11 @@ def post(self, user, post_key, comment_id=None, reply_id=None):
         """This method is only meant to give like in post."""
         post = ndb.Key(urlsafe=post_key).get()
         institution = post.institution.get()
+        current_institution_key = self.request.get('currentInstKey')
```

Comment:
Why not just pass the institution name? This approach it's going to be expensive, once we have to get the institution from the database then get the name. A good idea could be, pass a JSON institution just with the name, because, if we need to extend the data passed through the request it's not necessary to change the data structure.